docs/File-System-API.md

```diff
@@ -58,14 +58,69 @@ OutStreamOptions options = new OutStreamOptions.Builder(ClientContext.getConf())
 FileOutStream out = tfs.getOutputStream(path, options);
 ```
 
+### IO Options
+
+There are two different types of storage in Tachyon, Tachyon managed storage and under storage.
+Tachyon managed storage is the memory, SSD, and/or HDD which is allocated to Tachyon workers. For
+read and write operations, users can specify the `TachyonStorageType` to fine tune the interaction
+with Tachyon's managed storage. Likewise, users can specify the `UnderStorageType` to fine tune the
+data operations with the under storage, such as S3, Swift, or HDFS. Note that read operations are
+affected by `TachyonStorageType` (the data can be stored in more Tachyon workers) but not
+`UnderStorageType`.
+
+Below is a table of commonly used `TachyonStorageType` and `UnderStorageType` combinations.
+
+<table class="table">
```

Comment:
The two are independent (aside from the fact that they should not both be no-ops). The reason to break them down instead of combine them is to make it easier to understand how the data will interact with Tachyon and the underlying storage.